azure/Guidelines.md

```diff
@@ -747,18 +747,66 @@ The ability to retry failed requests for which a client never received a respons
 <!-- Open API Spec -->
 [OpenAPI Specification]: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md
 
-### Long Running Operations & Jobs
+### Long-Running Operations & Jobs
 
-Azure generally follows the [Microsoft REST API guidelines for Long running operations](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#13-long-running-operations).  Follow the Microsoft guidelines with the modifications / extensions given here when designing / implementing long running operations.
+When the processing for an operation may take a significant amount of time to complete, it should be
+implemented as a _long-running operation (LRO)_. This allows clients to continue running while the
+operation is being processed. The client determines the outcome of the operation at some later time
```

Comment:
@mikekistler -- I'm reacting to the word "determines" in this sentence. When I first read it, it sounded like the client can affect the outcome of the operation through an API call at a later time. What about something like...

```suggestion
operation is being processed. The client discovers the outcome of the operation at some later time
```